src/main/java/com/kulik/airbnb/service/UserService.java

```diff
@@ -29,6 +32,15 @@ public User get(int id) {
         return user;
     }
 
+    public String confirmUser(String token) throws Exception {
+        if (jwtTokenProvider.validateToken(token)) {
```

Comment:
looks like a security issue - a user can confirm an account using any valid token, but not necessarily the one that was sent by email